pandas/tseries/resample.py

```diff
@@ -208,6 +208,12 @@ def _assure_grouper(self):
         """ make sure that we are creating our binner & grouper """
         self._set_binner()
 
+    def plot(self, *args, **kwargs):
+        # for compat with prior versions, we want to
+        # have the warnings shown here and just have this work
+        return _maybe_process_deprecations(self, how='mean').plot(*args,
```

Comment:
no, he DID use ``mean`` implicity. Oh you want a more meaningful warning? ok sure